docs/source/release/index.rst

```diff
@@ -12,9 +12,7 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`release-pre-1.0`
 
-* :support:`2304` Update ``google-cloud-bigquery`` dependency minimum version to 1.12.0
```

Comment:
pls dont' edit the whatsnew except for additions. (except for a dedicated PR)

Reply:
Sorry about that didn't mean to delete those